docs/scenario-dashboard.md

```diff
@@ -195,19 +193,19 @@ in a scene.
       <img class='wide_img' src="../imgs/dashboard_simulator.png"/>
     </div>
  
-  1. Run a [Unit Under Test](/scenario-dashboard/#connecting-your-code).
+  1. Run a [Unit Under Test](../scenario-dashboard/#connecting-your-code).
```

Comment:
Shouldn't this go after configuring the batch?

Reply:
This can go before or after configuring the batch. To me, it make more sense after opening and playing the simulator.